From f4997d4cb92ade203ec5ba4f9a3cb5d0c65e8ee4 Mon Sep 17 00:00:00 2001
From: Gwenael Treuveur <gwenael.treuveur@foss.st.com>
Date: Fri, 22 Nov 2024 17:43:10 +0100
Subject: [PATCH 1387/1695] dts: stm32: add OEM rproc public key hash for
 stm32mp23

Add new OTP entry for remoteproc authentication key hash.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 048181972811de62452454b5b67918a0e196d2f8 dts: stm32: add OEM rproc public key hash for stm32mp23]
Change-Id: Id343052a595a4d7ef483b64d7fe97553d3f1e69d
Signed-off-by: Gwenael Treuveur <gwenael.treuveur@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/420443
Tested-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Gwenael TREUVEUR <gwenael.treuveur@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 core/arch/arm/dts/stm32mp231.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp231.dtsi b/core/arch/arm/dts/stm32mp231.dtsi
index 78dee4da3..39280ce55 100644
--- a/core/arch/arm/dts/stm32mp231.dtsi
+++ b/core/arch/arm/dts/stm32mp231.dtsi
@@ -2775,6 +2775,12 @@
 				reg = <0x240 0x20>;
 			};
 
+			oem_rproc_pkh: oem_rproc_pkh@2c0 {
+				/* 0x2C0 -> otp176 */
+				reg = <0x2C0 0x20>;
+				st,non-secure-otp-provisioning;
+			};
+
 			oem_fip_enc_key: otp260@410 {
 				reg = <0x410 0x20>;
 				st,non-secure-otp-provisioning;
-- 
2.47.2

